brd-android/app/src/main/AndroidManifest.xml

```diff
@@ -43,6 +43,7 @@
         android:launchMode="singleTask"
         android:networkSecurityConfig="@xml/network_security_config"
         android:supportsRtl="true"
+        tools:replace="android:theme"
```

Comment:
The clash with checkout frames on this theme must be fixed in a different way since is breaking the app theme (toolbar)